devinfo/src/mountinfo/bytebuf.rs

```diff
@@ -1,6 +1,7 @@
 use std::io::{IoSliceMut, Read};
 
-/// This is io::Read capable Sized type. This can wrap around a Vec<u8>.
+/// This is io::Read capable Sized type. This wraps around a Vec<u8>.
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub struct ByteBuf {
```

Comment:
Does this need to be pub?

Reply:
Made this private. Also removed the `#[derive(Clone, Debug, Eq, PartialEq)]` as these are not required for this crate's use.